.travis.yml

```diff
@@ -77,6 +77,22 @@ jobs:
       script:
         - NODE=$(which node) make lint lint-py
 
+    - name: "Docs"
+      if: type = pull_request
+      language: node_js
+      node_js: "node"
+      install:
+        - pyenv global 2.7.15
+        - make doc-only
+        - npm install -g linkinator
+      script:
+        - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
```

Comment:
Redundant because of 
https://github.com/nodejs/node/blob/3ed6684323a70d785494816a2e8601e76a34bf3e/.travis.yml#L81